content/en/logs/guide/collect-google-cloud-logs-with-push.md

```diff
@@ -16,7 +16,17 @@ further_reading:
 ---
 
 <div class="alert alert-danger">
-This page describes deprecated features with configuration information relevant to legacy Pub/Sub Push subscriptions, useful for troubleshooting or modifying legacy setups. Use a <strong>Pull</strong> subscription with the Datadog Dataflow template to forward your Google Cloud logs to Datadog instead. See <a href="https://docs.datadoghq.com/integrations/google_cloud_platform/#log-collection" target="_blank">Log collection</a> on the Google Cloud integration page for instructions.
+
+This page describes deprecated features with configuration information relevant to legacy Pub/Sub Push subscriptions, useful for troubleshooting or modifying legacy setups.
+
+Collecting Google Cloud logs with a Pub/Sub Push subscription is in the process of being deprecated for the following reasons:
+
+- If you have a Google Cloud VPC, new Push subscriptions cannot be configured with external endpoints (see Google Cloud's [Supported products and limitations][12] page for more information)
+- The Push subscription does not provide compression or batching of events, and as such is only suitable for a very low volume of logs
```

Comment:
If the reasoning info can't be removed:

```suggestion
This page describes deprecated features with configuration information relevant to legacy Pub/Sub Push subscriptions, useful for troubleshooting or modifying legacy setups. Pub/Sub Push subscription is being deprecated for the following reasons:
- For Google Cloud VPC, new Push subscriptions cannot be configured with external endpoints (see Google Cloud's [Supported products and limitations][12] page for more information)
- The Push subscription does not provide compression or batching of events, and is only suitable for low volume of logs
```

Reply:
Please remove "is only suitable for low volume of logs"

Reply:
Thanks for clarifying on the reasoning @sriramkraman! I'll leave it in (with the low volume of logs removed)